### ISSUE TITLE
Mekanism gas issues

### ISSUE THREAD

**Issue by freopt** [shedaniel/RoughlyEnoughItems#725]
**Describe the bug**
Mekanism gasses dont load in REI,
I was testing out the Mekanism ore processing to see if it was actually broken or if REI was broken, then crash

**Steps to Reproduce**
Load game
No Mekanism gasses

**Screenshots**

**Environment (please complete the following information with the version):**
 - Minecraft: 1.16.5
 - Mod Loader: Forge, using workaround

**Logs**
Pastebin the log [e.g. `logs/latest.log`] and link it below:
crash: https://www.toptal.com/developers/hastebin/amequkimon.rb
basically 2/3 of my log were these errors repeated: https://www.toptal.com/developers/hastebin/azeladufah.properties

**Comment by shedaniel** [comment]
Can I get the full log?

**Comment by freopt** [comment]
file was too big for any pastebin https://drive.google.com/file/d/1ca9zFQF-N5wSxX0b-jF9_5nyo5dflGZL/view?usp=sharing
